docs/UserGuide.md

```diff
@@ -59,6 +59,11 @@ CCACommander Ultra Promax Xtra 9000PLUS is the one-stop app for CCA Heads to man
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </div>
 
+### Clear all Members and Events: `clear`
+Clears all Member and Event entries from CCACommander.
```

Comment:
Should this also mention clearing all attendances, or is there no need since it can be inferred all attendances are gone?